### PR TITLE
feat: support wildcard certificate

### PR DESCRIPTION
**Pull request by j-zimnowoda**
implements: https://app.zenhub.com/workspaces/dev-5e7e84dddc966f05a627a19b/issues/gh/redkubes/otomi-core/1134

depends-on: https://github.com/redkubes/otomi-console/pull/315
depends-on: https://github.com/redkubes/otomi-api/pull/437

### CODE REVIEW

File: versions.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-api: 0.10.0
-console: 0.11.0
+api: wildcard-cert
```
Comment (ElderMatt) [inline versions.yaml:1]: Do these need to be changed back?

**Review by ElderMatt — CHANGES_REQUESTED** [review]

File: versions.yaml (hunk 1 above)
Comment (j-zimnowoda) [reply versions.yaml:1]: After api and console is merged I will change it.

**Review by ferruhcihan — APPROVED** [review]
LGTM

**Review by ElderMatt — APPROVED** [review]
LGTMT

**Pull request merged**